release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -139,6 +139,18 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-auth"]
 === Authentication and authorization
 
+[id="ocp-4-11-auth-oidc-providers"]
+==== Active Directory Federation Services is now a supported OIDC provider
+
+The Active Directory Federation Services OpenID Connect (OIDC) provider is now tested and supported with {product-title}.
+
+[NOTE]
+====
+Currently, it is not supported to use Active Directory Federation Services with {product-title} when custom claims are used.
```

Comment:
To be more elaborate, would it be better to link https://issues.redhat.com/browse/RFE-2770 (it is public) and/or https://bugzilla.redhat.com/show_bug.cgi?id=2072927?

Reply:
@xingxingxia Normally I wouldn't mind linking to a BZ or Jira, but it looks like those have both been closed/rejected. So I don't know if those would be good to link to. It sounds like we're not going to change anything on our end for this, but instead hope that Microsoft will. wdyt?

Reply:
Yes, their closed/rejected statuses make it awkward to link them, though they indeed include elaborate info which may be useful. Anyway, let's not link then. Thx